app/javascript/styles/rtl.scss

```diff
@@ -112,6 +112,16 @@ body.rtl {
     margin-right: 6px;
   }
 
+  .fa-ul {
+    margin-left: 0;
+    margin-left: 2.14285714em;
```

Comment:
ahhhh... so fa is font-awesome, not something specific to the Farsi language. OK :-)